src/modules/launcher/PowerLauncher/Storage/QueryHistory.cs

```diff
@@ -4,11 +4,11 @@
 using Newtonsoft.Json;
 using Wox.Plugin;
 
-namespace Wox.Storage
+namespace PowerLauncher.Storage
 {
-    public class History
+    public class QueryHistory
     {
-        public List<HistoryItem> Items { get; set; } = new List<HistoryItem>();
+        public List<HistoryItem> Items { get; } = new List<HistoryItem>();
```

Comment:
[clarification] Why did we have to remove the setter?

Reply:
`Items` variable was never set. Static analysis requires that this property should be read-only which can be done by removing setter.